bridges/NordbayernBridge.php

```diff
@@ -88,6 +97,18 @@ private function getUseFullContent($rawContent) {
 		return $content;
 	}
 
+	private function getTeaser($content) {
+		$teaser = $content->find('p[class=article__teaser]', 0);
+		if($teaser == null) {
+			return '';
+		}
+		$teaser = $teaser->plaintext;
+		$teaser = preg_replace('/ {143}- {66}/', ' - ', $teaser);
+		$teaser = preg_replace('/ {53}/', '', $teaser);
```

Comment:
Combine the preg_replace calls:
```suggestion
		$teaser = preg_replace('/[ ]{2,}/', ' ', $teaser);
```